Readme flesh 2

jlouis opened the pull request:
Re-ordering of the sections to make it more readable.

yrashk commented:
It looks like this change negates my recent changes to README.md. Can you please update it, if possible?

jlouis committed:
Re-order the sections in the README.md file.

This is hopefully easier for a user to comprehend. We quickly
introduce the commands needed by most users to run the commands. Then
we go on to tell all the details. The assumption is that most users
will simply ask for "How do I fetch package X?".

jlouis commented:
Oops, repushed. Emacs, you should have been nice!
